src/cascadia/WindowsTerminal_UIATests/Common/PgoManager.cs

```diff
@@ -9,16 +9,25 @@ namespace WindowsTerminal.UIA.Tests.Common
 {
     public static class PgoManager
     {
+        private static string TrimExtension(string str)
+        {
+            if (str.EndsWith(".exe") || str.EndsWith(".dll"))
+            {
+                return str.Substring(0, str.Length - 4);
+            }
+            return str;
+        }
+
         public static void PgoSweepIfInstrumented(TestContext context, string assemblyName)
         {
-#if PGO_INSTRUMENT
+#if true

```

Comment:
This should probably be changed to `#if PGO_INSTRUMENT` or removed entirely.

Reply:
Woopsie. I should put this back. Thanks.